src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -740,14 +740,16 @@ def create_ag_request_routing_rule(cmd, resource_group_name, application_gateway
         url_path_map=SubResource(id=url_path_map) if url_path_map else None)
     if cmd.supported_api_version(min_api='2017-06-01'):
         new_rule.redirect_configuration = SubResource(id=redirect_config) if redirect_config else None
+    if cmd.supported_api_version(min_api='2019-04-01'):
```

Comment:
what's practice to set min_api version? I thought it's for azure stack. is it possible to specify version at global level, instead of in business logic code? it's hard for maintenance.